rickandmorty/src/components/atoms/Button/style.module.scss

```diff
@@ -0,0 +1,23 @@
+@import 'styles/mixins.module.scss';
+@import 'styles/variables.module.scss';
+
+.button {
+    height: 2.325rem;
+    background: $bc;
+    border-radius: 0.25rem;
+    padding: 0 1rem;
+    outline: 0;
+    border: 0;
+    color: $white;
+    transition: background-color 100ms ease-in-out;
+    cursor: pointer;
+
+    @media screen and (max-width: 500px) {
```

Comment:
`@mixin mobile () {
   @media screen and (max-width: 500px) {
        @content
}
}`